compiler/runtime/Helpers.inc

```diff
@@ -50,7 +50,8 @@ SETVAL(TR_writeBarrierBatchStore,TR_writeBarrierStoreRealTimeGC+1)
 SETVAL(TR_arrayStoreException,TR_writeBarrierBatchStore+1)
 SETVAL(TR_throwClassCastException,TR_arrayStoreException+1)
 SETVAL(TR_throwCurrentException,TR_throwClassCastException+1)
-SETVAL(TR_IncompatibleClassChangeError,TR_throwCurrentException+1)
+SETVAL(TR_throwUnreportedException,TR_throwCurrentException + 1)
```

Comment:
What does this line have to do with the rest of the commit that seems to be more about marking and checking a synthetic exception handler ?

Reply:
It's for adding the VM helper for throwing exceptions from the synthetic exception handler. It's also part of eclipse/openj9#9267.

Reply:
@vijaysun-omr can you resolve the discussion if this answer is acceptable?

Reply:
Yes I was just wondering why this change is there when it is not mentioned in the commit or PR message as such. I obviously have no issue with adding the helper.

Reply:
Added description for the helper int the PR description.